src/diffusers/utils/hub_utils.py

```diff
@@ -564,7 +564,8 @@ def push_to_hub(
             commit_message (`str`, *optional*):
                 Message to commit while pushing. Default to `"Upload {object}"`.
             private (`bool`, *optional*):
-                Whether or not the repository created should be private.
+                Whether to make the repo private. If `None` (default), the repo will be public unless the
+                organization's default is private. This value is ignored if the repo already exists.
```

Comment:
Maybe link to the appropriate docs for enterprise orgs?

Reply:
I'd rather keep it simple (+ a bit lazy to update the 5 other PRs I've opened on other HF libraries :smile:)